pom.xml

```diff
@@ -74,7 +90,7 @@
     <plugins>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-        <version>3.7.0</version>
```

Comment:
Interesting that this plugin got downgraded. Is this just what your IDE produced? Or did you manually downgrade this one?

Reply:
Good catch! This is the default version used by my IDE's app engine plugin. I bumped it to 3.8.1 which is the latest and everything appears fine.